google/cloud/bigtable/batcher.py

```diff
@@ -54,19 +145,49 @@ class MutationsBatcher(object):
     flush. If it reaches the max number of row mutations size it calls
     finish_batch() to mutate the current row batch. Default is MAX_ROW_BYTES
     (5 MB).
+
```

Comment:
Please remove the TODO from the doc
Also please document the threadsafety behavior. From what I see in the code, a Batcher can't be shared by multiple enduser  threads?

Reply:
I've removed the TODO. Correct the same batcher cannot be shared by multiple enduser threads.